test/utils/file_utils.py

```diff
@@ -16,9 +16,12 @@
 import tarfile
 
 
-def make_tarfile(script, model, output_path, filename="model.tar.gz"):
+def make_tarfile(script, model, output_path, filename="model.tar.gz", script_path=None):
     output_filename = os.path.join(output_path, filename)
     with tarfile.open(output_filename, "w:gz") as tar:
-        tar.add(script, arcname=os.path.basename(script))
+        if(script_path):
+            tar.add(script, arcname=os.path.join(script_path, os.path.basename(script)))
+        else:
+            tar.add(script, arcname=os.path.basename(script))
```

Comment:
assuming `script_path` is defaulted to empty string:

```suggestion
        tar.add(script, arcname=os.path.join(script_path, os.path.basename(script)))
```
this is because `os.path.join` with empty string arg has [no effect on the resulting path](https://github.com/python/cpython/blob/3.8/Lib/posixpath.py#L76-L86):

```
>>> os.path.join("", "/tmp/foo")
'/tmp/foo'
>>> os.path.join("", "/tmp", "foo")
'/tmp/foo'
>>> os.path.join("", "tmp", "foo")
'tmp/foo'
```